builtin/providers/docker/config.go

```diff
@@ -0,0 +1,24 @@
+package docker
+
+import dc "github.com/fsouza/go-dockerclient"
```

Comment:
Nitpick I usually just do multiline imports from the get-go cause there is usually some future that requires more and it makes it easier to add.